daemon/volumes_unix.go

```diff
@@ -389,6 +389,14 @@ func (daemon *Daemon) createVolume(name, driverName string, opts map[string]stri
 	return v, nil
 }
 
+func removeVolume(v volumedrivers.Volume) error {
```

Comment:
Shouldn't need this, there is already a function to do this, and it needs to be gated by the ref counter.